src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.html

```diff
@@ -1,12 +1,54 @@
 <cd-orchestrator-doc-panel *ngIf="showDocPanel"></cd-orchestrator-doc-panel>
-<cd-table *ngIf="hasOrchestrator"
-          #daemonsTable
-          [data]="daemons"
-          [columns]="columns"
-          columnMode="flex"
-          [autoReload]="5000"
-          (fetchData)="getDaemons($event)">
-</cd-table>
+
+<div  *ngIf="flag === 'hostDetails'; else serviceDetailsTpl">
+  <cd-table *ngIf="hasOrchestrator"
+            #daemonsTable
+            [data]="daemons"
+            [columns]="columns"
+            columnMode="flex"
+            [autoReload]="5000"
+            (fetchData)="getDaemons($event)">
+  </cd-table>
+</div>
+
+<ng-template #serviceDetailsTpl>
+  <ng-container>
+    <ul ngbNav
+        #nav="ngbNav"
+        class="nav-tabs"
+        cdStatefulTab="service-details">
+      <li ngbNavItem="details">
+        <a ngbNavLink
+           i18n>Details</a>
+        <ng-template ngbNavContent>
+          <cd-table *ngIf="hasOrchestrator"
+                    #daemonsTable
+                    [data]="daemons"
+                    [columns]="columns"
+                    columnMode="flex"
+                    [autoReload]="9000"
```

Comment:
Why 9 secs (I guess that's it, right)?

On the other hand, I see many tables (e.g.: `pool/pool-list/pool-list.component.html`), set `[autoReload]="-1"` or `false`... Maybe that approach is more efficient that letting the table self-refresh it and trigger the fetch method?

I'd suggest you (if you didn't give it a try yet) to [run `ng.profiler.timeChangeDetection({record : true})`](https://dmitrymogilko.medium.com/profiling-angular-change-detection-c00605862b9f) from the JS console.